src/eos/eosEngine.js

```diff
@@ -548,7 +553,70 @@ export class EosEngine extends CurrencyEngine {
             return { server, result }
           }
         )
-        out = await asyncWaterfall([...hyperionFuncs])
+        const dfuseFuncs = this.currencyInfo.defaultSettings.otherSettings.eosDfuseServers.map(
+          server => async () => {
+            if (this.currencyInfo.currencyCode !== 'EOS')
+              throw new Error('dfuse only supports EOS')
+            const response = await this.eosJsConfig.fetch(`${server}/graphql`, {
+              method: 'POST',
+              headers: {
+                'Content-Type': 'application/json'
+              },
+              body: JSON.stringify({
+                query: dfuseGetTransactionsQueryString,
+                variables: {
+                  query: `${
+                    direction === 'outgoing' ? 'auth' : 'receiver'
+                  }:${acct} action:transfer`,
+                  limit,
+                  low
+                }
+              })
+            })
+            const responseJson = asEither(
+              asDfuseGetTransactionsResponse,
+              asDfuseGetTransactionsErrorResponse
+            )(await response.json())
+            if (responseJson.errors != null) {
+              this.log.warn(
+                `dfuse ${server} get transactions failed: ${JSON.stringify(
+                  responseJson.errors[0]
+                )}`
+              )
+              throw new Error(responseJson.errors[0].message)
+            }
+            // Convert txs to Hyperion
+            const actions = responseJson.data.searchTransactionsBackward.results.map(
+              tx =>
+                asHyperionTransaction({
+                  trx_id: tx.trace.id,
+                  // Remove the Z (2020-02-07T02:47:22.5Z => 2020-02-07T02:47:22.5)
+                  '@timestamp': tx.trace.block.timestamp.slice(
+                    0,
+                    tx.trace.block.timestamp.length - 1
+                  ),
```

Comment:
Don't remove the "Z". Do a bit of discovery with date parsing on hyperion responses to see if there is issues with localization of the time string.